README.md

```diff
@@ -1,13 +1,13 @@
 # dotnet script
 
-Run C# scripts from the .NET CLI.
+Run C# scripts from the .NET CLI.  [![Nuget](http://img.shields.io/nuget/v/dotnet-script.svg?maxAge=10800)](https://www.nuget.org/packages/dotnet-script/)
```

Comment:
Would it make sense to list the other packages as well?
* Dotnet.Script.Core
* Dotnet.Script.DependencyModel

Maybe formatted in a table like the build status?

Reply:
The `Dotnet.Script` package is soon to be deprecated and the `Dotnet.Script.DependencyModel.NuGet` is sort of just a transitive so these does not need to be listed

Reply:
I added all for now, after all ,they are public 😄 
once we deprecate one of them, I think it's even better to mention it here so that it's not confusing when someone searches nuget